src/main/java/lv/acodemy/homework/Variables.java

```diff
@@ -0,0 +1,347 @@
+package lv.acodemy.homework;
+
+public class Variables {
+    public static void main(String[] args) {
+
+        String task1 = "\nTask 1: Create 10 variables for each data type\n";
+        System.out.println(task1);
+
+        String byte1 = "Task 1.1. byte\n";
+        System.out.println(byte1);
+
+        byte price1 = 100;
```

Comment:
Price of what? Variable does not make sense at all, please make it more meaningful :)))

Reply:
Fixed